examples/openvino/image-classification/README.md

```diff
@@ -39,4 +39,31 @@ python run_image_classification.py \
     --save_total_limit 3 \
     --seed 1337 \
     --output_dir /tmp/beans_outputs/
+```
+
+### Structured Pruning on Swin for food101
+Following is an example to prune Swin-base model structurally while fine-tuning for food101. More on how to configure the pruning algorithm, see NNCF documentation [here](https://github.com/openvinotoolkit/nncf/blob/develop/nncf/experimental/torch/sparsity/movement/MovementSparsity.md).
```

Comment:
Would it be possible to add how long this took to train on one GPU, and what the resulting accuracy is, like in the other JPQD examples?

Reply:
Yes, it's WIP.

Reply:
see [here](https://github.com/vuiseng9/optimum-intel/tree/pr-nncf-jpqd/examples/openvino/image-classification#joint-pruning-quantization-and-distillation-on-swin-for-food101)